desktop/mac/index.md

```diff
@@ -127,29 +127,28 @@ File share settings are:
 
 #### Proxies
 
-Docker Desktop detects HTTP/HTTPS Proxy Settings from macOS and automatically
-propagates these to Docker. For example, if you set your
-proxy settings to `http://proxy.example.com`, Docker uses this proxy when
-pulling containers.
+HTTP/HTTPS proxies can be used when
 
-If you want to configure proxies manually, turn on the **Manual proxy configuration** setting.
+- logging in to Docker
```

Comment:
Nit pick: can we capitalize the first word of each bullet point to align with our style guide?